recipe/meta.yaml

```diff
@@ -16,15 +16,19 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3.7
     - pip
+    - jupyter-packaging
   run:
-    - python
+    - python >=3.7
     - pygments >=2.4.1,<3
 
 test:
   imports:
     - jupyterlab_pygments
+  commands:
+    - test -f ${PREFIX}/share/jupyter/labextensions/jupyterlab_pygments/package.json                            # [unix]
```

Comment:
Yes this is already a good way to test whether the extension prebuilt assets are in the archive :+1: